### PR TITLE
Update config.xml

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -265,6 +265,7 @@
 <!-- ALP -->
   <doc cat="alp" doc="alp-dolomite" branches="main">The SUSE ALP Dolomite Guide</doc>
   <doc cat="alp" doc="cockpit-alp-dolomite" branches="main">The SUSE ALP Dolomite Cockpit Guide</doc>
+  <doc cat="alp" doc="deploying-alp-dolomite-interactively-using-agama" branches="main">The SUSE ALP Dolomite Cockpit Guide</doc>
```
Comment (tomschr) [inline config.xml:268]: The title is wrong, shouldn't it be "Deploying ALP Dolomite interactively using Agama"?

**Review by tomschr — CHANGES_REQUESTED** [review]
Thanks, just the title and it's okay, right?

File: config.xml (hunk 1 above)
Comment (tbazant) [inline config.xml:268]:
```suggestion
  <doc cat="alp" doc="deploying-alp-dolomite-interactively-using-agama" branches="main">Deploying ALP Dolomite interactively using Agama</doc>
```

**Review by tbazant — COMMENTED** [review]
fixed a title

**Review by tomschr — APPROVED** [review]

**Pull request merged**